source/frontend/src/models/api/UserOverrideCode.ts

```diff
@@ -3,6 +3,7 @@ export enum UserOverrideCode {
   ADD_LOCATION_TO_PROPERTY = 'ADD_LOCATION_TO_PROPERTY',
   UPDATE_REGION = 'UPDATE_REGION',
   PROPERTY_OF_INTEREST_TO_INVENTORY = 'PROPERTY_OF_INTEREST_TO_INVENTORY',
+  CONTRACTOR_NOT_IN_TEAM = 'CONTRACTOR_NOT_IN_TEAM',
```

Comment:
Both of the below two override codes seem suspect, as a user should not be able to override not being in the team or removing themselves.

Reply:
Removed